.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: ruby
+rvm:
+  - 2.5.3
+
+addons:
+  code_climate:
+    repo_token: 89943dcf9eece00c7e360730a4a6102b1ce29c1b9956186bf4801ad059169274
+
+env:
+  global:
+    - RUBY_GC_MALLOC_LIMIT=90000000
+    - RUBY_GC_HEAP_FREE_SLOTS=200000
+    - SECURITY_SECRET_TOKEN=traviscitraviscitraviscitraviscitravisci
+
+services:
+  - postgresql
+  #- redis-server
```

Comment:
We usually remove the line instead of leaving it commented (the same for the others below)